packages/devextreme/js/__internal/grids/new/grid_core/data_controller/types.ts

```diff
@@ -1,3 +1,9 @@
 export type DataObject = Record<string, unknown>;
 export type Key = unknown;
 export type KeyExpr = unknown;
+export interface OperationOptions {
+  filtering?: boolean;
+  sorting?: boolean;
+  paging?: boolean;
+  summary?: boolean;
```

Comment:
Let's remove summary as we will not support it in this iteration. It is okay to remove it only from public type so that customers will not be confused